contrib/pp_osx_codesign_fix/pp_osx_codesign_fix.c

```diff
@@ -0,0 +1,96 @@
+/* -*- C -*- main.c */
+
```

Comment:
Please add a simple copyright and license note, e.g. the first 4 paragraphs of the LICENSE file
would do (of course at your disgression, but don't forget to change the copyright).